demos/taco-demo/src/App.tsx

```diff
@@ -32,7 +32,7 @@ export default function App() {
 
   useEffect(() => {
     initialize();
-    switchNetwork(Mumbai.chainId);
+    switchNetwork(80002);
```

Comment:
I guess there may not be support for an 'Amoy' object just yet?

Reply:
Yes, we need to bump a bunch of deps that may expose this object but it's going to cause a mess. Hence going to address it at another time.

Reply:
👍